app/services/taxonomy/taxon_fetcher.rb

```diff
@@ -1,11 +1,9 @@
 module Taxonomy
   class TaxonFetcher
-    # Return a list of taxons from the publishing API with links included.
     def taxons
-      @taxons ||=
-        Services.publishing_api.get_linkables(document_type: 'taxon')
-          .map { |taxon_hash| Taxon.new(taxon_hash) }
-          .sort_by(&:title)
+      @taxons ||= taxon_list.sort do |taxon_1, taxon_2|
```

Comment:
I think it might be better to use the publishing-api's ordering method instead of doing it here. That would enable us to use pagination later.